e2e/tests-dfx/call.bash

```diff
@@ -55,6 +55,17 @@ teardown() {
   assert_match '("Hello, Names are difficult!")'
 }
 
+@test "call subcommand accepts raw argument from stdin" {
```

Comment:
In this test, `dfx canister call` isn't reading the raw argument from stdin.  Am I missing something?